development/nginx/data/example.yml

```diff
@@ -5,12 +5,7 @@ vars:
   _insights_signature_exclude: "/vars/insights_signature,/vars/content_vars"
   content: |
     #!/bin/sh
-    data='{"alert": false, "summary": "convert2rhel did not detect issues", "report": "", "report_json": {"foo": "bar"}}'
-    /usr/bin/convert2rhel --help
-    /usr/bin/convert2rhel --version
-    echo "BEGIN MARKER"
-    echo "$data"
-    echo "END MARKER"
+    echo "Hello, world!"
```

Comment:
Do we want to change the README as well?

Reply:
Good point. I will update the README.md